BuildingSync.xsd

```diff
@@ -14155,9 +14155,11 @@ Window spacing: the dimension between windows in a discrete window layout. (in.)
       <xs:restriction base="xs:string">
         <xs:enumeration value="Property management company"/>
         <xs:enumeration value="Corporation/partnership/LLC"/>
-        <xs:enumeration value="Religious organization"/>
+        <xs:enumeration value="Privately owned"/>
```

Comment:
@markborkum -- you requested `privately owned school` but the building type shouldn't be part of the ownership, so I just made it `privately owned`